meta-resin-common/recipes-core/initrdscripts/files/ro-rootfs

```diff
@@ -1,3 +1,7 @@
 #!/bin/sh
 
 bootparam_ro="true"
+
+ro_enabled() {
```

Comment:
Just tested on my laptop and it doesn't work. `bash: ro-rootfs_enabled: command not found`